front_end/html/navbar_exercise.html

```diff
@@ -16,6 +16,9 @@
                                     <img src="/static/unchecked.png" alt="Not completed" class="navbar-img"/>
                                 {% end %}
                                 {{ exercise[1]["title"] }}
+                                {% if exercise[1]["enable_pair_programming"] %}
```

Comment:
I mentioned this in an email, but please add a space after the title and before the P.

Reply:
Please also make sure the text won't wrap if we have a longer title (or the screen is narrow).

Reply:
Just to clarify, this is how the text currently wraps under worst case conditions. (this is the longest possible title and the narrowest screen I could make) Do you want me to make the text flow off the screen to the right?
<img width="318" alt="image" src="https://user-images.githubusercontent.com/61362908/120364120-17002580-c2ca-11eb-8157-468600f62944.png">

Reply:
This looks good. I like that it wraps. Not to be a pain, but it appears that the text is aligned vertically with the bottom of the status icon to the left of the text. Is there a way to center it vertically?

Reply:
Yes, I just fixed it!